HowManyBalloonsAlg.py

```diff
@@ -0,0 +1,15 @@
+"""How Many Balloons Alghorithm"""
+
+import math
+
+class BalloonCount(object):
+    def __init__(self, weight):
+        self.weight = weight
+
+    def how_many_balloons(self):
+        balloons = self.weight * 453.593 / 14
```

Comment:
Make both `453.593` and `14` class level constants like so

```
class blah:
    x = 453.593
    y = 14
```
but instead of x and y use the names of the variables, what they represent